src/antsibull_core/vendored/_argparse_booleanoptionalaction.py

```diff
@@ -3,6 +3,9 @@
 # It is licensed under the Python Software Foundation License:
 # https://github.com/python/cpython/blob/main/LICENSE
 #
+# SPDX-FileCopyrightText: 2001-2020 Python Software Foundation
```

Comment:
Where does this copyright date come from? It's different than the one in the PSF license file.

Reply:
The code was last modified in 2020 (https://github.com/python/cpython/commit/41b223d29cdfeb1f222c12c3abaccc3bc128f5e7).

Reply:
:+1: